lib/common/helpers.ts

```diff
@@ -567,6 +567,33 @@ export function stringify(value: any, replacer?: (key: string, value: any) => an
 	return JSON.stringify(value, replacer, space || 2);
 }
 
+export function getFormattedDate(): string {
```

Comment:
`getFormattedDate` in which format? As far as I see it makes the date strings orderable and it could be something like getOrderableDateString. Also, we could make it accept the Date as an argument in order to make it unit testable.